### ISSUE TITLE
 onnxruntime::ConcatBase::PrepareForComputeNon concat axis dimensions must match: Axis 2 has mismatched dimensions of 39 and 40

### ISSUE THREAD

**Issue by ouening** [onnx/models#392]
# Ask a Question

### Question
I try to use `tiny yolov3` model， but I got error:
```
2020-10-23 21:33:12.7990686 [E:onnxruntime:, sequential_executor.cc:318 onnxruntime::SequentialExecutor::Execute] Non-zero status code returned while running Concat node. Name:'concatenate_1' Status Message: Not satisfied: dim_value == inputs_0_dims[axis_index]
concat.cc:79 onnxruntime::ConcatBase::PrepareForComputeNon concat axis dimensions must match: Axis 2 has mismatched dimensions of 39 and 40
Segmentation fault
```

### Further information
python:3.7.4
pytorch: 1.6.0+cpu
onnx: 1.7.0
onnxruntime: 1.5.2

Is this issue related to a specific model?  
**Model name** (tiny yolov3):  
**Model opset** (ONNX v6, keras2onnx 1.6.0, ai.onnx v11):

### Notes
My code is simple:
```python
def infer_tiny_yolov3(onnx_model:str):
    
    # Run the model on the backend
    session = onnxruntime.InferenceSession(onnx_model, None)
    
    # get the name of the first input of the model
    input_name = session.get_inputs()[0].name  
    output_name = session.get_outputs()[0].name  
    # print(len(session.get_outputs()))
    print('Input Name:', input_name)
    print('Output Name:', output_name)
    
    # 符合https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/tiny-yolov3 模型的输入要求
    input_data, raw_img = preprocess(img_file, 618, 618)
    print('输入数据大小：', input_data.shape)
    print('输入数据类型：', input_data.dtype)
    image_size = np.array([raw_img.shape[1], raw_img.shape[0]], dtype=np.float32).reshape(1, 2)
    print('输入图像大小：', image_size.shape)
    start = time.time()
    raw_result = session.run([], {input_name: input_data,
                                  'image_shape':image_size})
    end = time.time()
    print('推理时间：', end-start,'s')
    
infer_tiny_yolov3(tiny_yolov3_onnx_model)
```

**Comment by ouening** [comment]
My mistake, the input shape should be divisible by 32， like 608

**Comment by devaale** [comment]
@ouening This gonna sound extra stupid - but how to figure out what is the number (in your case 32) that input shape should be divisible by ?
